src/hook/mypage/useDeleteJob.ts

```
@@ -0,0 +1,18 @@
+import { useMutation } from '@tanstack/react-query';
+import api from '@hook/api.ts';
+
+export const useDeleteJob = () => {
+  return useMutation({
+    mutationFn: (jobIds: number[]) =>
+      api.delete('/v1/member/job', { data: { jobIds } }),
+    onSuccess: (response) => {
+      if (response.data.success) {
+      } else {
```

ESLint

[error] Empty block statement.

(no-empty)